### PR TITLE
doc(apple search ads): Edit source README for Fivetran style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Part of https://fivetran.atlassian.net/browse/RD-187057

Fixes grammar errors. Rewords for clarity and style. Please see my question inline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,8 +41,8 @@ vars:
...
+### Disable models
+It's possible that your Apple Search Ads connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Apple Search Ads or actively excluded some tables from your syncs. To disable the corresponding functionality in the package, you must add the relevant variables. By default, the package assume that all variables are true. Add variables for only the tables you want to disable. 
 
 The `apple_search_ads_using_search_terms` variable below refers to the `search_terms_report` table. The [search match](https://searchads.apple.com/help/campaigns/0006-understand-search-match) functionality must be turned on within specified ad groups in order for there to be data populated in this table.
```
Comment (fivetran-chloe) [inline README.md]: @fivetran-joemarkiewicz I don't understand what we mean by "specified ad groups" here. Which specified ad groups?

File: README.md [hunk 2]
```diff
@@ -9,16 +9,16 @@ This package enriches your Fivetran data by doing the following:
 - Models staging tables, which will be used in our transform package
 
 ## Models
-This package contains staging models, designed to work simultaneously with our [Apple Search Ads modeling package](https://github.com/fivetran/dbt_apple_search_ads).  The staging models:
+This package contains staging models, designed to work simultaneously with our [Apple Search Ads modeling package](https://github.com/fivetran/dbt_apple_search_ads).  The staging models are designed to:
```
Comment (fivetran-chloe) [inline README.md]: @fivetran-joemarkiewicz In line 9 above, we say "transform package," but here we say "modeling package." I assume this refers to the same thing. I think we should pick one term and use it throughout. (I can work on applying this change to other READMEs once we make a decision.)

File: README.md (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply README.md]: Hey @fivetran-chloe the Apple Search Ads UI allows for you to create multiple ad groups and you must enable the `search_terms` functionality within each distinct ad group.

We are trying to highlight the fact that you must enable `search_terms` within each ad group if you want the data to be populated.

File: README.md (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply README.md]: Great call out. I agree we should be consistent. My opinion is to stick with "Modeling". This allows us to also align with the "Data Models" term we are using for product promotions.

Comment (fivetran-chloe) [reply README.md]: @fivetran-joemarkiewicz Our [new transformations docs](https://beta.fivetran.com/docs/transformations/dbt/data-models) call these "transform models." Could that work for the READMEs?

File: README.md (hunk 1 above)
Comment (fivetran-chloe) [reply README.md]: Got it - thanks! I've reworded this sentence for clarity - what do you think?

Comment (fivetran-chloe) [inline README.md]:
```suggestion
The `apple_search_ads_using_search_terms` variable below refers to the `search_terms_report` table. You must enable the [search match](https://searchads.apple.com/help/campaigns/0006-understand-search-match) function within each ad group to populate this table with data. 
```

Comment (fivetran-joemarkiewicz) [reply README.md]: Looks great, thanks!

File: README.md (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply README.md]: Yeah that would work perfect!

Comment (fivetran-chloe) [reply README.md]: Great, thanks! I've made the change here created a follow-up issue for our docs team to make this change throughout the rest of the READMEs: https://fivetran.atlassian.net/browse/RD-190128

Comment (fivetran-chloe) [inline README.md]:
```suggestion
This package contains staging models, designed to work simultaneously with our [Apple Search Ads transform package](https://github.com/fivetran/dbt_apple_search_ads).  The staging models are designed to:
```

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Thanks so much for applying these changes @fivetran-chloe. They look great!

**Pull request merged**